code/modules/clothing/glasses/glasses.dm

```diff
@@ -835,3 +835,15 @@ BLIND     // can't see anything
 	else
 		flash_protection = FLASH_PROTECTION_NONE
 		tint = TINT_NONE
+
+/obj/item/clothing/glasses/whiteblindfold
```

Comment:
Why is this not just a subtype of `/obj/item/clothing/glasses/sunglasses/blindfold`?

Reply:
got it